core/src/main/java/gov/nasa/jpl/mudrod/metadata/pre/ApiHarvester.java

```diff
@@ -53,6 +53,7 @@ public ApiHarvester(Properties props, ESDriver es, SparkDriver spark) {
   public Object execute() {
     LOG.info("Starting Metadata harvesting.");
     startTime = System.currentTimeMillis();
+    es.deleteType(props.getProperty("indexName"), props.getProperty("raw_metadataType"));
```

Comment:
Metadatalinkage type is removed before processing metadata
https://github.com/mudrod/mudrod/blob/master/core/src/main/java/gov/nasa/jpl/mudrod/utils/LinkageTriple.java#L78